language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/QNameRefCompletionUtil.java

```diff
@@ -109,7 +108,7 @@ public static List<Symbol> getModuleContent(PositionedOperationContext context,
                 QNameRefCompletionUtil.getAlias(qNameRef));
         return module.map(moduleSymbol -> moduleSymbol.allSymbols().stream()
                 .filter(predicate)
-                .collect(Collectors.toList()))
+                .toList())
```

Comment:
Same as above

Reply:
Fixed in [2778660](https://github.com/ballerina-platform/ballerina-lang/pull/42904/commits/277866060d74b08115089625bb7d582c095697a1)